README.md

```diff
@@ -1,2 +1,135 @@
 # lvl_is_flite
 Voice generation for Icelandic Flite voices
+
+## Preparation
+This section describes where the data for this voice comes from and how it is preprocessed.
+
+### Voice data
+This repo is intended for use with data from [the Talrómur corpus](https://repository.clarin.is/repository/xmlui/handle/20.500.12537/104), and in particular the first voice; lvl_is_f is created using voice f, named Álfur therein.
+There are 2 primary voice-specific files in this voice build. They are, relative to the voice directory: 
+- `etc/txt.done.data`, which contains normalized prompts with inserted phrasing tags
+- `festvox/lexicon.scm` which contains a mix of manual and automatic g2p transcriptions of all words that occur in the source data.
+
+For the initial voice, [the Grammatek TTS frontend](https://github.com/grammatek/tts-frontend) was used with phrasing enabled, but as of yet this is a very time consuming step. The phrasing step can optionally be skipped to greatly speed things up, in which case pause tags are only inserted where punctuation is present in the prompts.
+
+Due to serious limitations of Flite in terms of handling phoneme names, we use a modified version of SAMPA, which does not use underscores, digits or colons.
+
+<details>
+<summary>See mapping from standard SAMPA to our modified version</summary>
+
+|SAMPA|flite phoneset|
+|-----|--------------|
+|a:   |aa            |
+|a    |a             |
+|ai:  |aii           |
+|ai   |ai            |
+|au:  |auu           |
+|au   |au            |
+|c    |c             |
+|C    |C             |
+|c_h  |ch            |
+|D    |D             |
+|ei:  |eii           |
+|ei   |ei            |
+|E:   |EE            |
+|E    |E             |
+|f    |f             |
+|G    |G             |
+|h    |h             |
+|i:   |ii            |
+|i    |i             |
+|I:   |II            |
+|I    |I             |
+|j    |j             |
+|k_h  |kh            |
+|k    |k             |
+|l    |l             |
+|l_0  |lz            |
+|m    |m             |
+|m_0  |mz            |
+|n    |n             |
+|n_0  |nz            |
+|J    |J             |
+|J_0  |Jz            |
+|N    |N             |
+|N_0  |Nz            |
+|9:   |oee           |
+|9    |oe            |
+|9i:  |oeii          |
+|9i   |oei           |
+|ou:  |ouu           |
+|ou   |ou            |
+|O:   |OO            |
+|Oi   |Oi            |
+|O    |O             |
+|p_h  |ph            |
+|p    |p             |
+|r    |r             |
+|r_0  |rz            |
+|s    |s             |
+|t_h  |th            |
+|t    |t             |
+|u:   |uu            |
+|u    |u             |
+|v    |v             |
+|x    |x             |
+|Y:   |YY            |
+|Yi   |Yi            |
+|Y    |Y             |
+|T    |T             |
```

Comment:
isn't `pau` an official symbol ? Or is this phrasing already ? But isn't it trained with the current voice as some special phone ?